### DISCUSSION TITLE
CFAST results vs NUREG-1805 FDT for a room with forced ventilation

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/cfast#2060] [Q&A] [answered]
Hello all,

I am trying to model a compartment (10m x 10m x 4m) with 2000kW fire and forced ventilation with rate of 0.2m3/s for supply at floor level and exhaust at ceiling level in CFAST. Compartment is made of concrete (walls are 0.2m thick and ceiling /floor are 0.4m thick). I am looking for the time when the temperature reaches 500C. This does not happen in CFAST as the max temperature is reached by 350s and the value is around 250C, and after that the temperature begins to fall and fire power also decreases. When the compare such time and temperature with NUREG FDT tool (foote et al. method) for forced ventilation, such temperature using FDT for forced ventilation compartment is reached by 2028s. How can there be such a big difference? Or, I am wrong somewhere?

Please guide, thanks!

![6](https://github.com/firemodels/cfast/assets/146167548/64135585-6380-4923-a676-231e850e5961)
![5](https://github.com/firemodels/cfast/assets/146167548/13617e4d-eeb1-40ce-bd0a-214f247f8d82)
![4](https://github.com/firemodels/cfast/assets/146167548/8fc65907-0667-4b69-86b6-e5ffe6a4b302)
![3](https://github.com/firemodels/cfast/assets/146167548/e2f7931e-f67d-42e0-8b27-da3c20dff988)
![2](https://github.com/firemodels/cfast/assets/146167548/d05914ec-e920-4a62-aea0-66af8d771a9f)
![1](https://github.com/firemodels/cfast/assets/146167548/c8032a7b-d9db-429c-8476-449343f2cb0a)

**Comment by drjfloyd** [comment]
The FDTs are simple hand calculations and may not be applicable in all situations. You should always consider the assumptions behind correlations and how suitable they are for a specific scenario.

A 10 x 10 x 4 m compartment with no ventilation has about 110 kg of O2 in it.  The default lower O2 limit in CFAST is 15 % meaning you can consume about 28 kg of the O2 before you have extinction.  At 13100 kJ/kg of O2 that is ~360,000 kJ of heat release rate which at a constant 2000 kW is about 3 minutes.  0.2 m3/s of ventilation is ~0.08 kg/s of O2 of which you can consume ~0.02 kg/s or sustain 40 kW of HRR.

**Comment by Ranhill** [comment]
Hello drjfloyd!

Thanks for the reply! I understand that the fire will soon become ventilation limited due to oxygen deficiency. However, based on the mechanical ventilation with volumetric flow rate of  0.2 m3/s, I had calculated the max_HRR = 3000 kJ/(kg air) * volumetric_flow_rate (m3/s) * 1.2 (kg air/m3) = 760 kW. Such calculation is based on the assumption: For most fuels, the fire will	produce	a hrr of approximately	13,100 kJ/kg of oxygen or 3,000kJ/kg of airflow that is	available (Tewarson,2008). Can you please elaborate how you arrived at 40 kW using 0.08 kg/s? Is it not 0.04 kg/s of O2?

**Comment by drjfloyd** [comment]
The default CFAST lower oxygen limit is 15 %.

**Comment by Ranhill** [comment]
Thanks!

**Comment by Ranhill** [comment]
Hello again djfloyd!

I was verifying the calculation with lower oxygen limit to calculate the ventilation limited fire as HRR_ventilation_limited = 13100 * volumetric flow rate * 1.2 (0.21 -0.15) where volumetric flow rate is 0.2m3/s, 1.2 is air density, and O2 percentage for combustion range from 21% to 15%. The calulation amounts to 188.64 kW, where your calculation leads to 40 kW! Can you please guide where its getting wrong? Thanks!

**Reply by drjfloyd** [reply, marked as answer]
I'll sharpen the pencil a little, may have had a typo when doing the quick estimate:

0.2 m3/s * 1.2 kg air/m3 = 0.24 kg air/s
O2 has an ambient mass fraction of 0.23 kg O2/kg air
0.23 kg O2/kg air * 0.24 kg/s = 0.055 kg O2/s 
A lower limit of 15 % means you can burn from 21 % to 15 % or (21 %-15 %)/21 % =  28.6 % of the O2 can be consumed
0.055 kg O2/s * 28.6 % = 0.016 kg O2/s
0.016 kg O2/s * 13100 kJ/kg O2 = 206 kW

**Reply by Ranhill** [reply]
Thank you very much, drjfloyd!
